### PR TITLE
Add an option to GAPID to disable unknown extensions.

### PR DESCRIPTION
**Pull request by AWoloszyn**

### CODE REVIEW

File: gapii/cc/spy_base.h [hunk 1]
```diff
@@ -185,6 +185,8 @@ class SpyBase {
   track_memory::MemoryTracker mMemoryTracker;
 #endif  // TARGET_OS
 
+  // This is true if we should not show disabled extensions to the application.
```
Comment (pmuetschard) [inline gapii/cc/spy_base.h]: nit: this comment was hard for me to parse :) I think it be better phrased as "If true, we will hide disabled extensions from the application."

**Comment by AWoloszyn** [comment]
@ben-clayton  Thanks, I hadn't run the presubmits locally because this was done on my windows machine. I will pull on Linux and fix that.

File: gapii/cc/spy_base.h (hunk 1 above)
Comment (AWoloszyn) [reply gapii/cc/spy_base.h]: Sure

**Comment by AWoloszyn** [comment]
@pmuetschard  According to Ben, GLES doesn't advertise unknown extensions, so this should be a NO-op

I will open a bug to make the option... optional. (Similar to disablePCS)

**Pull request merged**